store/src/main/java/com/nytimes/android/external/store3/base/impl/CacheFactory.java

```diff
@@ -58,4 +59,50 @@ private CacheFactory() {
                     .build();
         }
     }
+
+    public static <Key, Parsed> Cache<Key, Observable<Parsed>> createRoomCache(MemoryPolicy memoryPolicy) {
```

Comment:
needed a new cache implementation that holds observables rather than singles to keep the "live query" setup

Reply:
These are essentially the same as the single versions. You can keep it simple and have a method level generic for the singles/maybes/observables.

`private static <Key, Value> Cache<Key, Value> createBaseCache(MemoryPolicy memoryPolicy)`